charters/DataStore/charter.md

```diff
@@ -0,0 +1,72 @@
+
+# Data Store
+
+
+## Description
+The Data Store is a scientific data sharing/publishing/distribution framework, providing file/bundle management on multiple clouds at petabyte-scale. It defines public APIs for storage, retrieval, and subscription to events that functions transparently across multiple cloud systems such as AWS and GCP.
+
+## Objective
+The objective of the Data Store group is to deliver substantively complete functionality on all of the in-scope items listed in this charter.
+
+## In-scope
+
+### Interfaces
+* Data Store read and write APIs for data (PUT bundle, PUT file, GET bundle, GET file) - maintenance and extension of the implementation of the basic data access APIs.
+* Maintain and extend the **Checkout service API** which enables data checkout to a local filesystem or a personal cloud environment
```

Comment:
It is not clear what "checkout" is here.

Reply:
ah, I shouldn't reuse the word checkout here... Will do.